<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Aaron Ponti</author>
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>870</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lbDataSection">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Data folder</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pBSelectDataRootFolder">
     <property name="text">
      <string>Select data root folder</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlImageSelector">
     <item>
      <widget class="QSlider" name="hsImageSelector">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="tracking">
        <bool>false</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelImageCounterCurrent">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>7</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lbTrainingSection">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Training</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gLSplit">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="hLSplitFirstRow">
       <item>
        <widget class="QLabel" name="lbTrainingValidationSplit">
         <property name="text">
          <string>Training</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbValidationTestingSplit">
         <property name="text">
          <string>Validation/Test</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="hLSplitThirdRow">
       <item>
        <widget class="QLabel" name="lbNumberTrainingImages">
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbNumberValidationTestingImages">
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="hLSplitSecondRow">
       <item>
        <widget class="QSlider" name="hsTrainingValidationSplit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="hsValidationTestingSplit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlArchitecture">
     <item>
      <widget class="QComboBox" name="cbArchitecturePicker">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>Segmentation (U-Net 2D)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Restoration (U-Net 2D)</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbArchitectureSettings">
       <property name="text">
        <string>Settings</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pbTrain">
     <property name="text">
      <string>Train</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbPredictionSection">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Prediction</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gLPrediction">
     <item row="0" column="0">
      <widget class="QPushButton" name="pbSelectPredictionDataFolder">
       <property name="text">
        <string>Select data folder</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pbSelectPredictionTargetFolder">
       <property name="text">
        <string>Select target folder</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pbApplyPrediction">
       <property name="text">
        <string>Predict</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pbSelectPredictionModelFile">
       <property name="text">
        <string>Select model</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="pbFreeMemory">
     <property name="text">
      <string>Free GPU memory</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pBSelectDataRootFolder</tabstop>
  <tabstop>hsImageSelector</tabstop>
  <tabstop>hsTrainingValidationSplit</tabstop>
  <tabstop>hsValidationTestingSplit</tabstop>
  <tabstop>cbArchitecturePicker</tabstop>
  <tabstop>pbArchitectureSettings</tabstop>
  <tabstop>pbTrain</tabstop>
  <tabstop>pbSelectPredictionDataFolder</tabstop>
  <tabstop>pbSelectPredictionTargetFolder</tabstop>
  <tabstop>pbSelectPredictionModelFile</tabstop>
  <tabstop>pbApplyPrediction</tabstop>
  <tabstop>pbFreeMemory</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
